src/main/java/com/panemu/tiwulfx/control/dock/DetachableTabPane.java

```diff
@@ -198,9 +187,14 @@ private void attachListeners() {
 				if ( DRAG_SOURCE != DetachableTabPane.this ) {
 					final Tab selectedtab = DRAGGED_TAB;
 					DetachableTabPane.this.getTabs().add(dropIndex, selectedtab);
-					Platform.runLater(
-							() -> DetachableTabPane.this.getSelectionModel().select(selectedtab)
-					);
+					// Both are UI threads, there is no need to call Platform.runLater;
+					// If we need to wait for a period of time to execute, then we can use Pause Transition
+					//Platform.runLater(
+					//		() -> DetachableTabPane.this.getSelectionModel().select(selectedtab)
+					//);

```

Comment:
I think it would make more sense to get rid of the commented out old code.

Then update the comment explaining how `PauseTransition` resolves the issue where the tab could sometimes end up being small.